plugins/pom.xml

```diff
@@ -84,11 +84,16 @@ Portions Copyright (c) 2020, Chris Fraire <cfraire@me.com>.
             <artifactId>jersey-test-framework-provider-grizzly2</artifactId>
             <version>${jersey.version}</version>
             <scope>test</scope>
+            <exclusions>
```

Comment:
why this exclusion ? Does grizzly supply its own junit dependency ?

Reply:
Yes, I think so. The `JerseyTest` contains `@Before` and `@After` imports from JUnit4 and without this exclusion we could use the JUnit4 asserts. Hence, some of the changes in the test files.